src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
Hardcoding height for blocks with content inside - is a bad practice. please remove height